modules/ROOT/pages/manage-backup-restore.adoc

```diff
@@ -65,16 +65,14 @@ For OpenShift clusters, the following is backed up:
 
 === Changes Made After Performing a Backup Are Not Restored
 
-Any changes you make after you perform a backup will not be present if you restore the cluster from that backup. For example, if you perform a backup, delete an application, and then restore from that backup, the application you deleted will be recreated in the cluster, but it will be marked as deleted in Runtime Manager. As a result the application's status will show as *applying* in Runtime Manager.
+All changes performed on the control plane side do not need a new backup to be taken into account since those are taken from the platform at the time of restoration.
```

Comment:
```suggestion
All changes performed on the control plane side don't need a new backup to be taken into account because those are taken from the platform at the time of restoration.
```